src/poprox_concepts/domain/article.py

```diff
@@ -33,6 +35,7 @@ class Article(BaseModel):
     source: str | None = None
     external_id: str | None = None
     raw_data: dict[str, Any] | None = None
+    image: Image
```

Comment:
Since we want to store multiple images per article, I think we want this to be:
```suggestion
    images: list[Image]
```

Reply:
Updated and PR request sent. Thank you.